src/app/features/item/item-template/item-template.component.ts

```diff
@@ -58,7 +62,86 @@ export class ItemTemplateComponent extends SingleRowEditorComponent<ItemTemplate
   constructor(
     public editorService: ItemTemplateService,
     public handlerService: ItemHandlerService,
+    public sqliteQueryService: SqliteQueryService,
```

Comment:
actually we should use `public readonly`, I forgot it in the existing dependencies as well